llvm/test/tools/llvm-dwarfdump/X86/debug_info_crel.s

```diff
@@ -0,0 +1,288 @@
+# Test llvm-dwarfdump handles CREL relocation sections correctly.
```

Comment:
Can simplify with llvm/utils/update_test_body.py

Actually, instead of creating yet another complex test case, perhaps pick an existing one and add crel RUN

Reply:
I had to use a yaml test instead because clang is not a dependency of the test suite and if I use the assembly file (produced with clang -S), it doesn't generate the `.crel.debug_*` sections in the object file : https://github.com/llvm/llvm-project/commit/b3457d823e5e08462f7bb73b9c667da542e76a45